docs/root/operations/cli.rst

```diff
@@ -20,6 +21,17 @@ following are the command line options that Envoy supports.
   <https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format>`_
   formats respectively.
 
+.. option:: --config-yaml <yaml string>
+
+  *(optional)* The YAML string for a v2 bootstrap configuration. If --config-path is also set,
+   the values in this yaml string will override and merge with the ones --config-path.
```

Comment:
Nit: merge with the bootstrap loaded from --config-path

Reply:
Done.